namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-education-employment-dev/resources/irsa.tf

```diff
@@ -1,49 +0,0 @@
-locals {
-  sns_topics = {
-    "cloud-platform-Digital-Prison-Services-e29fb030a51b3576dd645aa5e460e573" = "hmpps-domain-events-dev"
-  }
-  sqs_queues = {
```

Comment:
This SNS config was configured purely to communicate with the `hmpps-education-and-work-plan-api` and is no longer required.